tests/integration/components/ice-dropdown-test.js

```diff
@@ -1,277 +1,189 @@
-  await waitForAnimations(dropdownHelpers.DROPDOWN_SELECTOR);
+  await dropdown.content.clickCloseItem();
 
-  assert.equal(dropdownHelpers.getCloseAttribute('[data-test-link]'), null,
-    'Link does not have close attribute');
+  assert.ok(!dropdown.isOpen, 'dropdown removed after clicking the close element');
 });
 
 test('dropdown box modifier class can be added', async function(assert) {
   assert.expect(1);
 
   this.render(hbs`
-    <div data-test-dropdown-target>
+    <div>
       Target
-      {{#ice-dropdown class="foobar"}}
+      {{#ice-dropdown data-test-dropdown=true class="foobar"}}
         template block text
       {{/ice-dropdown}}
     </div>
   `);
 
-  await dropdownHelpers.toggleDropdown('[data-test-dropdown-target]');
-  await waitForAnimations(dropdownHelpers.DROPDOWN_SELECTOR);
+  const dropdown = DropdownHelper.extend({
+    content: {
+      hasAdditionalClass: hasClass('foobar')
+    }
+  }).create();
 
-  assert.equal(dropdownHelpers.getDropdown().classList.contains('foobar'), true,
-    'dropdown box reflects additional class');
+  await dropdown.open();
+
+  assert.ok(dropdown.content.hasAdditionalClass, 'dropdown box reflects additional class');
 });
 
-test('dropdown box correctly renders content', async function(assert) {
+test('dropdown box direction can be modified', async function(assert) {
   assert.expect(1);
 
   this.render(hbs`
-    <div data-test-dropdown-target>
+    <div>
       Target
-      {{#ice-dropdown}}
+      {{#ice-dropdown data-test-dropdown=true placement="bottom-end"}}
         template block text
       {{/ice-dropdown}}
     </div>
   `);
 
-  await dropdownHelpers.toggleDropdown('[data-test-dropdown-target]');
-  await waitForAnimations(dropdownHelpers.DROPDOWN_SELECTOR);
+  const dropdown = DropdownHelper.create();
 
-  assert.equal(dropdownHelpers.getDropdown().textContent.trim(), 'template block text',
-    'dropdown renders given content');
+  await dropdown.open();
+
+  assert.equal(dropdown.content.placement, 'bottom-end', 'dropdown box reflects correct direction');
 });
 
-test('dropdown box direction can be modified', async function(assert) {
-  assert.expect(2);
+test('dropdown trigger element is marked as active when open', async function(assert) {
+  assert.expect(3);
 
   this.render(hbs`
-    <div data-test-dropdown-target>
+    <div>
       Target
-      {{#ice-dropdown placement="bottom-end"}}
+      {{#ice-dropdown data-test-dropdown=true placement="bottom-end"}}
         template block text
       {{/ice-dropdown}}
     </div>
   `);
 
-  await dropdownHelpers.toggleDropdown('[data-test-dropdown-target]');
-  await waitForAnimations(dropdownHelpers.DROPDOWN_SELECTOR);
-
-  assert.equal(dropdownHelpers.getDropdown().getAttribute('x-placement'), 'bottom-end',
-    'dropdown box reflects correct direction');
-  assert.equal(find('[data-test-dropdown-target]').getAttribute('dropdown-placement'), 'bottom-end',
```

Comment:
@LydiaBarnes we talked about this before but I couldn't remember if we decided it was necessary to keep or not. If it is I can try to figure out a way to add it to the component, but it'll be tricky since dropdowns can change placement when updated, etc.

Reply:
Yeah exactly, it was only kind of halfway working, since it only indicates the initial direction you manually give it. What I really wanted was for the arrow to follow the direction when its updated, say if the dropdown gets pushed over to a different area due to the window size. But is not a big deal if its too tricky.

Reply:
yeah, I think we should punt on it for now, we can always revisit later on